adapter/internal/oasparser/model/mgw_swagger.go

```diff
@@ -1221,6 +1222,9 @@ func (swagger *MgwSwagger) PopulateSwaggerFromAPIYaml(apiData APIYaml, apiType s
 	// context value in api.yaml is assigned as xWso2Basepath
 	swagger.xWso2Basepath = data.Context + "/" + swagger.version
 
+	// Enable/Disable Backend JWTs per api
+	swagger.DisableBackendJWT = data.DisableBackendJWT
```

Comment:
If this new property is not present we have to set the default value as true. Let's add test cases as well.

Reply:
We want backend JWT to be disabled by default? Ack on adding test cases.

Reply:
Added test case